Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/GeoTrackingSceneView.swift

```diff
@@ -0,0 +1,204 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ARKit
+import SwiftUI
+import ArcGIS
+
+/// A scene view that provides an augmented reality world scale experience using geo-tracking.
+public struct GeoTrackingSceneView: View {
+    /// A Boolean value indicating if the camera was initially set.
+    @Binding var initialCameraIsSet: Bool
+    /// The view model for the calibration view.
+    @ObservedObject private var calibrationViewModel: WorldScaleSceneView.CalibrationViewModel
+    /// The geo-tracking configuration for the AR session.
+    private let configuration = ARGeoTrackingConfiguration()
+    /// A Boolean value that indicates if the user is calibrating.
+    private let isCalibrationViewEnabled: Bool
+    /// The location datasource that is used to access the device location.
+    private let locationDataSource: LocationDataSource
+    /// The closure that builds the scene view.
+    private let sceneViewBuilder: (SceneViewProxy) -> SceneView
+    /// The proxy for the ARSwiftUIView.
+    @State private var arViewProxy = ARSwiftUIViewProxy()
+    /// The camera controller that will be set on the scene view.
+    @State private var cameraController: TransformationMatrixCameraController
+    /// A Boolean value that indicates whether the coaching overlay view is active.
+    @State private var coachingOverlayIsActive = false
+    /// The current camera of the scene view.
+    @State private var currentCamera: Camera?
```

Comment:
```suggestion
```
This property is updated to upon camera changes but is never read from. Can this be removed?

Reply:
Yes. I intended to use it to get the scene view's spatial reference so to project the vertical coordinate, but now I think it may be redundant.